addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -1158,13 +1158,14 @@ var MediaDialog = Dialog.extend({
 
         this.opened(function () {
             var tabToShow = 'icon';
+            var parentClassAttr = self.$media.parent().attr('class');
             if (!self.media || self.$media.is('img')) {
                 tabToShow = 'image';
             } else if (self.$media.is('a.o_image')) {
                 tabToShow = 'document';
             } else if (self.$media.attr('class').match(/(^|\s)media_iframe_video($|\s)/)) {
                 tabToShow = 'video';
-            } else if (self.$media.parent().attr('class').match(/(^|\s)media_iframe_video($|\s)/)) {
```

Comment:
Don't we want to use `self.$media.parent().hasClass('media_iframe_video')` ?

Reply:
I don't know the original reason to wrap the class with extra spaces, it really looks odd

Reply:
There is no wrapping the class with extra spaces... It checks that the classes contains the class 'media_iframe_video'. If you don't put the `(^|\s)` and `($|\s)`, you could match 'media_iframe_video_stuff'...

If you blame the line (which is the minimum when fixing that kind of line ?), you will find that this was not using jquery at first. Then jquery was used to get the parent... so the `hasClass` function can be used.